src/applications/disability-calculator/components/DisabilityRatingCalculator.jsx

```diff
@@ -50,11 +54,15 @@ export default class DisabilityRatingCalculator extends React.Component {
   handleSubmit = () => {
     const ratings = this.state.ratings;
     const calcRating = calculateRating(ratings);
-
-    this.setState({
-      showCombinedRating: true,
-      calculatedRating: calcRating,
-    });
+    const ratingArr = pullRatingsFromState(ratings);
+    const checkForTwoRatings = shouldCalculate(ratingArr);
+
+    if (checkForTwoRatings) {
```

Comment:
created `shouldCalculate` helper function so it will only calculate if there are at least two ratings.